dags/kpi_dashboard.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+
+from airflow import models
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    'owner': 'jklukas@mozilla.com',
+    'start_date': datetime.datetime(2019, 5, 12),
+    'email': ['telemetry-alerts@mozilla.com', 'jklukas@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'depends_on_past': False,
+    # If a task fails, retry it once after waiting at least 5 minutes
+    'retries': 1,
+    'retry_delay': datetime.timedelta(minutes=10),
+}
+
+dag_name = 'kpi_dashboard'
+
+with models.DAG(
+        dag_name,
+        # KPI dashboard refreshes at 16:00 UTC, so run this 15 minutes beforehand.
+        schedule_interval='45 15 * * *',
+        default_args=default_args) as dag:
+
+    kpi_dashboard = bigquery_etl_query(
+        task_id='kpi_dashboard',
```

Comment:
I don't think this should override the default of using `destination_table` for `task_id`
```suggestion
```